tokio-trace/src/subscriber.rs

```diff
@@ -38,6 +38,8 @@ pub trait Subscriber {
         value: &dyn IntoValue,
     ) -> Result<(), AddValueError>;
 
+    fn add_follows_from(&self, span: &span::Id, follows: span::Id);
```

Comment:
Taking the ID of the span to annotate by-reference may not really be necessary...